app/views/distributions/index.html.erb

```diff
@@ -28,28 +44,28 @@
           <p>
       </div>
     </div><!-- /.box-header -->
-  <div class="box-body">
-    <div class="row">
-      <div class="col-xs-12">
-          <!-- /.box-header -->
-          <div class="box-body table-responsive no-padding">
-            <table class="table table-hover striped">
-              <thead>
-                <tr>
-                  <th>Partner</th>
-                  <th>Date of Distribution</th>
-                  <th>Source Inventory</th>
-                  <th>Total items</th>
-                  <th>Total value</th>
-                  <th>&nbsp;</th>
-                </tr>
-              </thead>
-              <tbody>
-                <%= render partial: "distribution_row", collection: @distributions %>
-                <%= render partial: "distribution_total" %>
-              </tbody>
-            </table>
-          </div><!-- /.box-body.table-responsive -->
+    <div class="box-body">
+      <div class="row">
+        <div class="col-xs-12">
+            <!-- /.box-header -->
+            <div class="box-body table-responsive no-padding">
+              <table class="table table-hover">
```

Comment:
Could you re-add `striped` to this please? Michael added it earlier for a different issue

```suggestion
              <table class="table table-hover striped">
```

Reply:
done